src/main/java/SearchStockUseCase/ViewStockResponseModel.java

```diff
@@ -0,0 +1,4 @@
+package SearchStockUseCase;
+
+public class ViewStockResponseModel {
```

Comment:
I think you forgot to implement this class

Reply:
Good catch I removed the unused files now